### ISSUE TITLE
Tag Manager preview is causing a failed HTTP request `API.getPagesComparisonsDisabledFor`

### ISSUE THREAD

**Issue by tsteur** [matomo-org/matomo#19177]

**Comment by sgiehl** [comment]
@AltamashShaikh Yes. That one is triggered by core, but it can be disabled using an event, which needs to be added to TagManager.
See e.g. https://github.com/matomo-org/matomo/blob/cd541083f351a2712e58fa83a2c54861a92c120e/plugins/MultiSites/MultiSites.php#L18-L31
